erroranalysis/erroranalysis/_internal/surrogate_error_tree.py

```diff
@@ -429,13 +537,35 @@ def get_regression_metric_data(df):
 
 
 def get_classification_metric_data(df):
+    """Compute classification metric data from a dataframe.
+
+    :param df: dataframe
+    :type df: pandas.DataFrame
+    :return: pred_y, true_y, error
+    :rtype: numpy.array, numpy.array, int
+    """
     pred_y = df[PRED_Y]
     true_y = df[TRUE_Y]
     error = df[DIFF].values.sum()
     return pred_y, true_y, error
 
 
 def compute_metric_value(func, classes, true_y, pred_y, metric):
+    """Compute metric from the given function, true and predicted values.
+
+    :param func: The metric function to evaluate.
+    :type func: function
+    :param classes: List of classes.
+    :type classes: list
+    :param true_y: True y values.
+    :type true_y: numpy.array
+    :param pred_y: Predicted y values.
+    :type pred_y: numpy.array
```

Comment:
Is this numpy.ndarray or numpy.array? 

>>> import numpy as np
>>> np.ndarray
<class 'numpy.ndarray'>
>>> np.array
<built-in function array>

Seems like numpy.array is built-in function array.

Reply:
Looks like numpy.array is just the function but the actual type returned by this is numpy.ndarray.

Reply:
There are other places in error analysis SDK where we have numpy.array. It will be nice if we could fix those as a part of this PR.

Reply:
done, replaced numpy.array with numpy.ndarray everywhere